src/main/java/org/fundacionjala/at15/pokemon/io/Writer.java

```diff
@@ -22,9 +22,9 @@ public static void writeToJson(Entity entity) {
         }
     }
 
-    private static void checkFolder(File path) {
+    public static void checkFolder(File path) {
```

Comment:
Did you do a rebase before? because this change is already merged on dev/pokemon

Reply:
Rebase is not necessary with this change